soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -2520,7 +2521,7 @@ CustomMessage Randomizer::GetFishingPondOwnerMessage(u16 originalTextId) {
     auto ctx = Rando::Context::GetInstance();
     CustomMessage messageEntry = CustomMessage(
       "Sorry, but the pond is closed.&I've lost my good %rfishing pole%w...&Can't go fishing without it!",
-      "",
+      "Entschuldigung, aber der Teich ist geschlossen.&Ich habe meine gute %rAngelrute%w verloren...&Ohne kann ich nicht fischen!",
```

Comment:
Text overflows on geschlossen and the ...'s after verloren

Reply:
I modified this one a bit, but I tend to return to the initial translation "geschlossen" later, when this PR is merged. Temporarily, it seems to me that the shortened version with "zu" is the easier solution right now.